src/ngraph/op/experimental/generate_mask.hpp

```diff
@@ -53,8 +64,9 @@ namespace ngraph
             void validate_and_infer_types() override;
             Shape m_shape;
             element::Type m_element_type;
-            unsigned int m_seed;
-            double m_probability;
+            bool m_use_seed{false};
+            uint64_t m_seed{0};
+            double m_probability{0.0};
```

Comment:
Should the default value be 1.0 ?
I think, GenerateMask should have an argument to pass probability  instead of defaulting to 0 or 1

Reply:
The old constructor specifies everything. There's a second constructor now that takes no args and lets you set things after construction, to support other deserialization formats.